backend/bridgecommon/bridgecommon.go

```diff
@@ -143,25 +143,16 @@ func TriggerAuth() {
 	globalBackend.TriggerAuth()
 }
 
-// CancelAuth triggers an authentication canceled notification.
-func CancelAuth() {
-	mu.Lock()
-	defer mu.Unlock()
-	if globalBackend == nil {
-		return
-	}
-	globalBackend.CancelAuth()
-}
-
 // AuthResult triggers an authentication result notification
 // on the base of the input value.
-func AuthResult(ok bool) {
+func AuthResult(result string) {
```

Comment:
I think it's better if this is of type AuthResult, with the conversion happening in mobileserver.go's `AuthResult` wrapper, because the string constants are defined there and not here.